# Example of a control command message
float32 velocity
float32 steering_angle
